### ISSUE TITLE
Semi Fine age

### ISSUE THREAD

**Issue by achafetz** [noahbartlett/Me_One_Last_Kiss#10]
Does 15-17 (from OVC) this really count as 15-19 if there is no other half of a band to make it sum to 15-19?

https://github.com/noahbartlett/Me_One_Last_Kiss/blob/365e5ad71241bb4dcb4966ea46afb15973d3522a/R%20Files/03_age_disags.R#L56

**Comment by noahbartlett** [comment]
yes. Because anyone who is age 15-17 also falls into the category of 15-19. It's not about using that as its own age band, but rather ensuring that those in that age group are put into the correct age band.

**Comment by noahbartlett** [comment]
But I do see your point. Perhaps we need to have an OVC-specific fine age disag, the same way we have an OVC-specific coarse age band (+18/<18).
